osx/README.md

```diff
@@ -43,13 +41,14 @@ There are two ways to build the installer, automated or manually.
 
     * KA-Lite repo on `develop` branch, or the specified repo
     * PyRun version 2.7
-    * Assessment zip 
-1. You can optionally pass a ka-lite archive repo url as an argument in this format:
+    * Assessment zip
+
+1. You can optionally pass a ka-lite archive repo and assessment url as an argument like this sample:
 
-    > ./setup.sh "https://github.com/learningequality/ka-lite/archive/0.14.x.zip"
+    > ./setup.sh "https://github.com/learningequality/ka-lite/archive/0.15.x.zip" "https://learningequality.org/downloads/ka-lite/0.15/content/assessment.zip"
 
     This is useful if you want to try a different fork or branch on your build.
-    It defaults to the `develop` branch at "https://github.com/learningequality/ka-lite/archive/develop.zip".
+    It defaults to the `develop` branch at "https://github.com/learningequality/ka-lite/archive/develop.zip" if you don't specify the first argument. The second arguemnt will also defaults to https://learningequality.org/downloads/ka-lite/0.14/content/assessment.zip.
```

Comment:
Pls re-factor this, can you show two examples: 

1. using only one argument
2. then another example to use two arguments